lib/iris/coords.py

```diff
@@ -2775,6 +2779,366 @@ def xml_element(self, doc):
         return cellMethod_xml_element
 
 
+class Connectivity(_DimensionalMetadata):
+    """
+    A CF-UGRID topology connectivity, describing the topological relationship
+    between two lists of dimensional elements. One or more connectivities
+    make up a CF-UGRID topology - a constituent of a CF-UGRID mesh.
+
+    See: https://ugrid-conventions.github.io/ugrid-conventions
+
+    """
+
+    def __init__(
+        self,
+        indices,
+        cf_role,
+        long_name=None,
+        var_name=None,
+        attributes=None,
+        start_index=0,
+        element_dim=1,
```

Comment:
Let me catchup on understanding `element_dim` :+1:

Reply:
Agreed that we'll adopt the common metadata api `members` i.e., include `standard_name` and `units`.

Reply:
85b98c992188ee3e41d6b2a98139024090ad2231